api/tests/certification/course/integration/infrastructure/repositories/sco-certification-candidate-repository_test.js

```diff
@@ -92,14 +135,16 @@ describe('Certification | Course | Integration | Repository | SCOCertificationCa
       });
 
       // then
-      expect(candidatesIds).to.deep.equal([candidateId]);
+      expect(candidatesIds).to.deep.equal([candidateOfTheTargetedDivision.id]);
     });
 
     it('retrieves last candidates ordered by lastname and firstname', async function () {
       // given
       const sessionInThePastId = databaseBuilder.factory.buildSession({ publishedAt: '2020-01-01' }).id;
       const sessionId = databaseBuilder.factory.buildSession({ publishedAt: '2024-01-01' }).id;
       const anOrganizationId = databaseBuilder.factory.buildOrganization().id;
+
+      // Students
       const aOrganizationLearnerId = databaseBuilder.factory.buildOrganizationLearner({
```

Comment:
Commentaires pas nécessaires à garder je pense :) 

```suggestion
      const anOrganizationLearnerId = databaseBuilder.factory.buildOrganizationLearner({
```

Reply:
Done